docs/apis/losses/losses_cn.md

```diff
@@ -0,0 +1,255 @@
+简体中文 | [English](losses.md)
+# [paddleseg.models.losses](../../../paddleseg/models/losses)
+
+
+## [BCELoss](../../../paddleseg/models/losses/binary_cross_entropy_loss.py)
+```python
+class paddleseg.models.losses.BCELoss(
+            weight = None,
+            pos_weight = None,
+            ignore_index = 255,
+            edge_label = False
+)
+```
+
+> Binary cross entropy 损失函数的实现。
+
+### 参数
+* **weight**  (Tensor | str, optional): 对每个批数据元素的损失手动地重新调整权重。如果设定该参数，
+且若传入的是一个 1D 张量，则其尺寸为 `[N, ]`，其数据类型为 float32 或 float64；
+若传入的是一个 str，则值必须指定为 'dynamic'，以使在每轮迭代中根据二元交叉熵动态的计算权重。
+            *默认:``'None'``*
+* **pos_weight** (float|str, optional): 正样本的权重。若传入的是一个 str，则值必须指定为 'dynamic'，以使在每轮迭代中动态的计算权重。
+            *默认:``'None'``*
+* **ignore_index** (int64, optional): 指定一个在标注图中要忽略的像素值，其对输入梯度不产生贡献。当标注图中存在无法标注（或很难标注）的像素时，可以将其标注为某特定灰度值。在计算损失值时，其与原图像对应位置的像素将不作为损失函数的自变量。 *默认:``255``*
```

Comment:
The note in original code "Specifies a target value that is ignored and does not contribute to the input gradient" is not  friendly to beginners.